package.json

```diff
@@ -56,7 +56,7 @@
     ]
   },
   "engines": {
-    "node": ">= 12 <15"
+    "node": ">= 12 <=15"
```

Comment:
Had to allow Node 15 for M1 support, see:

https://github.com/pmmmwh/react-refresh-webpack-plugin/issues/259#issuecomment-733199810